src/main/java/org/sonatype/gradle/plugins/scan/nexus/iq/NexusIqScanTask.java

```diff
@@ -162,6 +166,16 @@ else if (PolicyAction.WARN == policyAction) {
     }
   }
 
+  private String buildUserAgent() {
+    return String.format("%s/%s (Java %s; %s %s; Gradle %s)",
```

Comment:
This format for user agent (including the plugin version, java version, OS and Gradle version) matches the one used for other integrations like Jenkins, Maven and the IDEA plugin.